### PR TITLE
Feature/logging

### PR DESCRIPTION
**Pull request by frank0631**
I liked your branch, extended a bit on your work. 
- I made the logLevelFlag an app-wide flag
- normalized the logging in the report module
- hardcoded the test to have 'debug' level logging

### CODE REVIEW

File: gocd/gocd_test.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ var conf *config.Config
 
 func init() {
 	conf = config.NewConfig()
-	conf.SetLogLevel("info")
+	conf.SetLogLevel("debug")
```
Comment (tanderegg) [inline gocd/gocd_test.go]: @frank0631 Can you actually do a quick commit to leave this as "info" for now?  We can add in the Debug option (through environment variables or whatnot) at a later point, I think.

File: report/report_test.go [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+	"testing"
+
+	"github.com/cfpb/rhobot/config"
+)
+
+var conf *config.Config
+
+func init() {
+	conf = config.NewConfig()
+	conf.SetLogLevel("debug")
```
Comment (tanderegg) [inline report/report_test.go]: Same here, let's leave as Info for now.

File: rhobot.go [hunk 3]
```diff
@@ -49,6 +49,7 @@ func main() {
 		Usage: "yaml file containing email distribution list",
 	}
 
+	app.Flags = []cli.Flag{logLevelFlag}
```
Comment (tanderegg) [inline rhobot.go:52]: I had thought this didn't work, but I'm glad it does, thanks!

**Comment by frank0631** [comment]
set the log level back to info

**Pull request merged**